examples/openlayers/filtered.html

```diff
@@ -4,7 +4,7 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.5.0/css/ol.css" type="text/css">
     <script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.5.0/build/ol.js"></script>
     <script src="https://unpkg.com/underscore@1.13.1/underscore-min.js"></script>
-    <script src="https://unpkg.com/flatgeobuf@3.7.1/"></script>
+    <script src="https://unpkg.com/flatgeobuf@3.7.1/dist/flatgeobuf-ol.min.js"></script>
```

Comment:
This was actually a bug in my website "build script", so I'm glad I had the opportunity for further scrutiny.